docker/install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -xeu
+
+# Install dependencies
+
+apt update
+apt install -y lsb-release apt-transport-https
+
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 379CE192D401AB61
+echo "deb https://dl.bintray.com/sociomantic-tsunami/dlang \
+    $(lsb_release -cs) release prerelease" >> /etc/apt/sources.list.d/dlang.list
```

Comment:
Wouldn't it make sense to start at least from `sociomantictsunami/base` image instead of plain `ubuntu` to avoid doing this in every project?

Reply:
Ideally, we will want to have separate base image for deployment purpose. Right now I use plain `ubuntu` so that we know better what exactly are apps dependencies - and avoid accidentally using stuff present in `base`.

Reply:
But the base image doesn't have much else than adding extra Ubuntu repos we need: https://github.com/sociomantic-tsunami/cachalot/blob/v5.x.x/docker/base

It only installs some extra development tools (like `jfrog` or `travis`) but no libraries. We could just create another image to have only the repos added and rename the current `base` to `devel` or something, but it might be a bit too much.

Reply:
When you think about image sizes, it does quite a lot:

https://github.com/sociomantic-tsunami/cachalot/blob/v5.x.x/docker/base#L18
https://github.com/sociomantic-tsunami/cachalot/blob/v5.x.x/docker/base#L47
https://github.com/sociomantic-tsunami/cachalot/blob/v5.x.x/docker/base#L50
https://github.com/sociomantic-tsunami/cachalot/blob/v5.x.x/docker/base#L81

Reply:
Then we should do the thin base image. I prefer to keep things DRY instead of having to copy&paste the adding of apt repos on every project.

Reply:
I'm fine to keep it like it is now for this PR, but please create an issue in cachalot so we don't forget and don't spread this way too much :)

Reply:
https://github.com/sociomantic-tsunami/cachalot/issues/58